setup.py

```diff
@@ -28,6 +28,16 @@
 with open('README.md', encoding='utf-8') as readme_file:
     readme = readme_file.read()
 
+extras_require = {
+    "ipython": ["ipython"],
+    "qserver": ["pyzmq"],
```

Comment:
This `pyzmq` usage is independent of quserver.  This should just be `'zmq_publisher'` or something like that.